src/utils/SafeDockerNetworkRemover.ts

```diff
@@ -0,0 +1,66 @@
+/*-
+ *
+ * Hedera Local Node
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import shell from 'shelljs';
+import { IS_WINDOWS, NETWORK_NAMES } from "../constants";
+
+/**
+ * Checks if the given string is a valid Docker network ID.
+ * A valid Docker network ID is a 12-character hexadecimal string.
+ *
+ * @param {string} id - The string to be validated as a Docker network ID.
+ * @returns {boolean} - Returns true if the string is a valid Docker network ID, false otherwise.
+ *
+ * @example
+ * const id = "89ded1eca1d5";
+ * console.log(isCorrectDockerId(id)); // Output: true
+ *
+ * @example
+ * const invalidId = "invalidID123";
+ * console.log(isCorrectDockerId(invalidId)); // Output: false
+ */
+const isCorrectDockerId = (id: string) => id.trim() !== '' && /^[a-f0-9]{12}$/.test(id);
+
+/**
+ * Provides utility methods for safe networks removal
+ */
+export class SafeDockerNetworkRemover {
+  /**
+   * Removes all the networks started by docker compose.
+   */
+  public static removeAll() {
+    for (const network of NETWORK_NAMES) {
+      this.removeByName(network)
+    }
+  }
+
+  /**
+   * Removes one single network by its name.
```

Comment:
This is technically not true, as it will loop through all networks that match that name filter and remove them, so for example an argument of `hedera` would remove all 3 networks created here, and if in the future we have a network named `hedera-mirror-node-2` for example that would be removed together with `hedera-mirror-node` when called with the `hedera-mirror-node` parameter

Reply:
@isavov You are correct. I have updated the comment and added a remark to the README file regarding the functionality of our script. Currently, it will remove all networks with the “hedera-” prefix.